File: 151002/Zaitseva/Lab1/SpringApi/src/main/java/by/bsuir/springapi/model/entity/Post.java
package by.bsuir.springapi.model.entity;

import by.bsuir.springapi.model.AbstractEntity;
import lombok.Getter;
import lombok.Setter;
import lombok.experimental.SuperBuilder;

@Getter
@Setter
@SuperBuilder
public class Post extends AbstractEntity {
    private Long issueId;
    
    private String content;
}

File: 151002/Krumkachev/Lab5/Discussion/PostEntity/Post.cs
using Discussion.Common;

namespace Discussion.PostEntity
{
    public class Post(string content, string country) : AbstractEntity
    {
        public string Country { get; set; } = country;
        public int IssueId { get; set; }
        public string Content { get; set; } = content;

        public Post() : this(string.Empty, string.Empty) { }

        public Post(int id, int issueId, string content) : this(content, string.Empty)
        {
            Id = id;
            IssueId = issueId;
        }

        public Post(int id, int issueId, string content, string country) : this(content, country)
        {
            Id = id;
            IssueId = issueId;
        }
    }
}
